auth/go.mod

```diff
@@ -0,0 +1,27 @@
+module github.com/fluxcd/pkg/auth
+
+go 1.22.4
+
+require (
+	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.12.0
+	github.com/Azure/azure-sdk-for-go/sdk/azidentity v1.7.0
+	github.com/onsi/gomega v1.33.1
+	k8s.io/utils v0.0.0-20240502163921-fe8a2dddb1d0
```

Comment:
Please get rid of this dependency, the Git package has to say free of Kubernetes deps.